test-utils/runtime-tester/run_scenario/src/main.rs

```diff
@@ -0,0 +1,30 @@
+use clap::{App, Arg};
+use runtime_tester::Scenario;
+
+use std::path::Path;
+use std::time::{Duration, Instant};
+
+fn main() {
+    let matches = App::new("Run scenario")
+        .arg(
+            Arg::with_name("scenario")
+                .long("scenario")
+                .required(true)
+                .takes_value(true)
+                .help("Scenario to run"),
+        )
+        .get_matches();
```

Comment:
We already use clap3 with derived macro for neard, so I suggest that we use derive approach for all other tools as well:

https://github.com/near/nearcore/blob/7399e0964c2d87529206f3f59d91ddb785feafb7/neard/src/cli.rs#L16-L26

It increases consitency and makes much easiter to add and document the options.